docs/rst/formalia/titlepage.rst

```diff
@@ -11,8 +11,19 @@
   :alt: eProsima
   :target: http://www.eprosima.com/
 
-*eProsima DDS Visualizer Plugin* is a graphical desktop application providing real-time 2D charts to visualize
+|ddsvisualizer| is a graphical desktop application providing real-time 2D charts to visualize
```

Comment:
This requires to include `alias.include`.
I would add it, but as this is the main page, I do not think it would be a problem if no aliases are used here.